x-pack/metricbeat/module/statsd/server/registry.go

```diff
@@ -287,7 +286,7 @@ func (r *registry) getOrNew(name string, tags map[string]string, new func() inte
 	tc, ok := r.metrics[tagsKey]
 	if !ok {
 		counter := new()
-		r.metrics[tagsKey] = map[string]*metric{name: &metric{
+		r.metrics[tagsKey] = map[string]*metric{name: {
```

Comment:
Anonymous field initialization can be a bit obscure. This should be easier to read and understand.

```suggestion
		r.metrics[tagsKey] = map[string]*metric{name: &metric{
```

Reply:
Ah, okay. In some cases, I agree with your point but here it's redundant and doesn't help much. The type `map[string]*metric` is explicitly mentioned right before. Also as all Go code should ideally conform with gofmt's standard and gofmt's simplification itself recommends this. 

Ref: https://pkg.go.dev/cmd/gofmt#hdr-The_simplify_command

```
An array, slice, or map composite literal of the form:
	[]T{T{}, T{}}
will be simplified to:
	[]T{{}, {}}
```